counterparty-core/counterpartycore/lib/messages/order.py

```diff
@@ -573,6 +573,10 @@ def parse(db, tx, message):
             min_btc_quantity = 0.001 * config.UNIT  # 0.001 BTC
             if util.enabled("btc_order_minimum_adjustment_1"):
                 min_btc_quantity = 0.00001 * config.UNIT  # 0.00001 BTC
+            if util.enabled("fix_min_btc_quantity", tx["block_index"]):
+                min_btc_quantity = int(
+                    D("0.00001") * D(1e8)
+                )  # else min_btc_quantity==1000.0000000000001
```

Comment:
0.00001 * 1e8 is the value in the comment. I think the comment can be removed because it makes me think an extra case was commented out.

Reply:
yeah right! Added that to explain why I use `D("0.00001")` instead `D(0.00001)`.